bump with amdahl latest commit and minor corrections (#503)
carpentries-incubator/hpc-intro: episodes/16-transferring-files.Rmd

## Transferring a Directory

To transfer an entire directory, we add the `-r` flag for "**r**ecursive"

> copy the item specified, and every item below it, and every item below those ...

Reviewer: ```suggestion
copy the item specified, and every item below it, and every item below those...
```

> copy the item specified, and every item below it, and every item below those ...

Reviewer: ```suggestion
copy the item specified, and every item below it, and every item below those...
```

## Transferring a Directory

To transfer an entire directory, we add the `-r` flag for "**r**ecursive":
This copies the item specified, and every item below it, and every item below

> those, and soon, until it reaches the bottom of the directory tree rooted at

Reviewer: ```suggestion
those, and so on, until it reaches the bottom of the directory tree rooted at
```

> you provided.

Reviewer: ```suggestion
This copies the item specified, and every item below it, and every item below
those, and so on, until it reaches the bottom of the directory tree rooted at the
folder name you provided.
```
